apps/heureka/src/components/Services/ServicePanel/ServicePanel.tsx

```diff
@@ -0,0 +1,181 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Juno contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from "react"
+import {
+  Panel,
+  Stack,
+  PanelBody,
+  ContentHeading,
+  DataGrid,
+  DataGridRow,
+  DataGridCell,
+  DataGridHeadCell,
+  Tooltip,
+  TooltipTrigger,
+  TooltipContent,
+  Button,
+  Pagination,
+  Badge,
+  Icon,
+  Message,
+} from "@cloudoperators/juno-ui-components"
+import { useFetchServiceImageVersions } from "../useFetchServiceImageVersions"
+import { capitalizeFirstLetter, truncateVersion } from "../../common/Helpers/helpers"
+import { EmptyDataGridRow } from "../../common/EmptyDataGridRow/EmptyDataGridRow"
+import { ServiceType } from "../Services"
+
+export type IssueCounts = {
+  critical: number
+  high: number
+  medium: number
+  low: number
+  none: number
+}
+
+type ServicePanelProps = {
+  service: ServiceType
+  isLoading?: boolean
+  onClose: () => void
+}
+
+export const ServicePanel = ({ service, isLoading = false, onClose }: ServicePanelProps) => {
+  const { loading, imageVersions, error, totalNumberOfPages, currentPage, goToPage, totalCount } =
+    useFetchServiceImageVersions({
+      serviceCcrn: service?.name || "",
+      pageSize: 8,
+    })
+
+  // Show total count except when loading and it's the first load for a service
+  const showTotalCount = !loading || imageVersions.length > 0
+
+  const formattedImageVersions = imageVersions.map((version) => ({
+    imageName: version.ccrn,
+    imageVersion: version.version,
+    issueCounts: version.issueCounts,
+    serviceName: service?.name,
+  }))
+
+  return (
+    <Panel
+      heading={
+        <Stack gap="2">
+          <span>{capitalizeFirstLetter(service.name)} Service Overview</span>
+        </Stack>
+      }
+      opened={true}
+      onClose={onClose}
+      size="large"
+    >
+      <PanelBody>
+        <Stack gap="2" distribution="between" alignment="center" className="mb-2">
+          <ContentHeading
```

Comment:
You shouldn't use the `ContentHeading` component for this. You already have a heading for the Panel above. Introducing a second heading of the same semantic weight is confusing. also you're repeating the service name again. My suggestion is to add the info with the image version count directly to the PanelHeading instead.